### PR TITLE
fix: value secret_ref shouldn't use state

### PR DESCRIPTION
**Pull request by delca85**
After #51 I went through the values implementation as I recalled I used `planmodifiers` there too and I realized we can get rid of them.

### CODE REVIEW

**Comment by delca85** [comment]
@johanneswuerbach removing the plan modifiers raises [this issue](https://github.com/humanitec/terraform-provider-humanitec/pull/53/checks#step:5:119) (which happens only on GA not if acceptance tests are ran locally).

File: internal/provider/resource_value.go [hunk 1]
```diff
@@ -125,19 +125,16 @@ func (r *ResourceValue) Schema(ctx context.Context, req resource.SchemaRequest,
...
 						MarkdownDescription: "Secret Store id. This can't be humanitec (our internal Secret Store). It's mandatory if ref is defined and can't be used if value is defined.",
 						Optional:            true,
 						Computed:            true,
-						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
 					},
 					"version": schema.StringAttribute{
 						MarkdownDescription: "Only valid if ref is defined. It's the version of the secret as defined in the target store.",
 						Optional:            true,
 						Computed:            true,
-						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
```
Comment (delca85) [inline internal/provider/resource_value.go:140]: We could actually keep the plan modifier for `store` and `ref` as they shouldn't change (if not `null`) but it's not the case for `version`.

**Comment by delca85** [comment]
> @johanneswuerbach removing the plan modifiers raises [this issue](https://github.com/humanitec/terraform-provider-humanitec/pull/53/checks#step:5:119) (which happens only on GA not if acceptance tests are ran locally).

I'm looking around to fix this, I found a similar issue with [this comment](https://github.com/hashicorp/terraform/issues/1830#issuecomment-99635000).

**Review by johanneswuerbach — APPROVED** [review]
🎉

**Pull request merged**